js/constants/config.js

```diff
@@ -7,7 +7,7 @@
 // https://vault-staging.brave.com for a dev build
 // http://localhost:3000 for production
 const vaultHost = process.env.VAULT_HOST || 'https://vault-staging.brave.com'
-const adHost = process.env.AD_HOST || '//cdn.brave.com'
+const adHost = process.env.AD_HOST || '//oip.brave.com'
```

Comment:
I'm pretty sure it is only used at runtime so you should be able to set it whenever